Building a hangman game (#30)
ember-learn/ember-blog: source/2019-03-30-the-ember-times-issue-91.md

---

> ## [Building a Hangman Game with Ember Octane](https://www.twitch.tv/videos/400723351) 🎬

Reviewer: Sorry so much discussion on this one! The build is failing because of alex, our language checker, see: https://github.com/ember-learn/ember-blog/blob/master/source/CONTRIBUTING.md and https://travis-ci.org/ember-learn/ember-blog/builds/512207696#L1307-L1310. 

I think you'll need to add `<!--alex ignore hangman-hangwoman-->` above every line/paragraph with `hangman`. You can download alex and run it locally to see though...

Reviewer: Added it here https://github.com/ember-learn/ember-blog/pull/30/commits/2f5a1a46bf19719073203e2989787dcddd79c685, checked locally, gonna merge to prevent merge conflicts / move forward on the main branch

> Mike North, together with his wife Lisa Huang did a [life stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use Ember Octane and the Glimmer components that come with it. 

Reviewer 2: I think we usually do names as Github handles like this:

```suggestion
[@mike-north](https://github.com/mike-north), together with his wife Lisa Huang did a [life stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use Ember Octane and the Glimmer components that come with it. 
```

Reviewer 3: It will be great to link to both of our github! Thanks :)

Author: I am :) But this just has not updated!

> Mike North, together with his wife Lisa Huang did a [life stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use Ember Octane and the Glimmer components that come with it. 

Reviewer 2: ```suggestion
Mike North, together with his wife Lisa Huang did a [live stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use [the Ember Octane preview](https://emberjs.com/editions/octane/) and the Glimmer components that come with it. 
```

> Husband and wife team [@mike-north](https://github.com/mike-north) and [@lisaychuang](https://github.com/lisaychuang) did a [live stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They use [the Ember Octane preview](https://emberjs.com/editions/octane/) and the Glimmer components that come with it. 

Reviewer 4: ```suggestion
Husband and wife team [@mike-north](https://github.com/mike-north) and [@lisaychuang](https://github.com/lisaychuang) did a [live stream](https://www.twitch.tv/videos/400723351) on how to build a simple hangman game. They used [the Ember Octane preview](https://emberjs.com/editions/octane/) and the Glimmer components that come with it. 
```

> This video is a nice introduction into the world of Ember Octane and Glimmer components and definitely shows the benefits of doing peer programming 😊.

Reviewer: I tend to say pair programming, but peer also works, up to you!
